### PR TITLE
#28: BitBucket: create a 'Begin with BitBucket' button

### PR DESCRIPTION
**Pull request by livelybug**

### CODE REVIEW

File: dist/content/index.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
     <nav>
         <ol>
             <li id="hook-in"><a href="/install/">Begin with GitHub</a></li>
+            <li id="hook-in-bitbuckit"><a href="/installBitbucket/">Begin with BitBuckit</a></li>
```
Comment (ScalaWilliam) [inline dist/content/index.html]: Spelling

File: conf/routes [hunk 2]
```diff
@@ -8,4 +8,5 @@ GET         /install/                      @controllers.github.InstallHooks.inst
 POST        /install/                      @controllers.github.InstallHooks.installPost
 GET         /static/*name                  @controllers.StaticDev.get(name: String)
 GET         /version/                      @controllers.Version.version
+GET         /installBitbucket/             @controllers.github.InstallHooks.installGetBitbucket
```
Comment (ScalaWilliam) [inline conf/routes]: Typically in REST preferable to have `/install-bitbucket/` and not camel case.

I'll take note in making this explicit in future.

File: app/controllers/github/InstallHooks.scala [hunk 3]
```diff
@@ -32,6 +32,11 @@ class InstallHooks @Inject()(wsClient: WSClient, configuration: Configuration,
       SeeOther(routes.InstallHooks.installGet().url).withSession("access-token" -> accessToken)
   }
 
+  //Burt, initial attempt
+  def installGetBitbucket = Action { req =>
+    Ok("not implemented")
```
Comment (ScalaWilliam) [inline app/controllers/github/InstallHooks.scala:37]: If anything, refer to the ticket number in the comment.

File: conf/routes (hunk 2 above)
Comment (livelybug) [reply conf/routes]: Google and found hyphen case is default best practice for URL

File: conf/routes [hunk 4]
```diff
@@ -8,5 +8,5 @@ GET         /install/                      @controllers.github.InstallHooks.inst
 POST        /install/                      @controllers.github.InstallHooks.installPost
 GET         /static/*name                  @controllers.StaticDev.get(name: String)
 GET         /version/                      @controllers.Version.version
-GET         /installBitbucket/             @controllers.github.InstallHooks.installGetBitbucket
+GET         /install-Bitbucket/            @controllers.github.InstallHooks.installGetBitbucket
```
Comment (ScalaWilliam) [inline conf/routes:11]: Typically - https://www.quora.com/Should-URLs-ever-have-capital-letters

**Pull request closed without merging**